src/bin/cli/mod.rs

```diff
@@ -515,20 +515,23 @@ impl PackageTrait for DocPackage {
 pub fn imports_arg<T: PackageTrait>(package: &T) -> FileResolver {
     let mut resolver = FileResolver::default();
 
-    for filename in package.get_input() {
-        if let Ok(path) = PathBuf::from(filename).canonicalize() {
-            let _ = resolver.add_import_path(path.parent().unwrap());
-        }
-    }
-
```

Comment:
Adding the parent of files as a basepath is incorrect IMO, and is not what solc does. I think we should remove this

Reply:
Yes, I agree. This should be removed. However, for some reason this is causing test failures in the `ethereum_solidity_tests` which exist exactly to compare behavior with solc. That needs debugging.